frontend/src/App.js

```diff
@@ -14,7 +14,7 @@ function App() {
       setLoading(true);
 
       axios
-        .get("http://localhost:5000/", { params: { query: searchQuery } })
+        .get("http://localhost:3000/", { params: { userName:'emanNahed', word: searchQuery} })
```

Comment:
you should keep the call to localhost:5000 because that is the backend! 

Also, do you need to hard code username? if the frontend is working correctly, you should be able to type in the searchbox and get that on the backend: { params: { query: searchQuery } }

searchQuery should represent the username!